SEED-platform/seed
# Return never-populated 'extra_data' columns for properties and tax lots

#### Any background context you want to provide?
Issue #1733 

#### What's this PR do?
Returns previously unpopulated `extra_data` columns when a property/ tax lot show API endpoint is called. This was done by updating PropertyState- and TaxLotState- Serializers.

#### How should this be manually tested?
  - Run server
  - Create a new column via API:
     Post the following to https://localhost:8000/api/v2/columns/:
```
{
  "shared_field_type": "None",
  "data_type": "string",
  "is_extra_data": true,
  "table_name": "PropertyState",
  "display_name": "Created from API Display Name",
  "merge_protection": "Favor New",
  "column_name": "Created from API"
}
```
  - Get the property data via the API endpoint:  
    `127.0.0.1:8000/api/v2/properties/<property_id>/?organization_id=<organization_id>`
  - Verify that the column does appear 

#### What are the relevant tickets?
Issue #1733 

#### Screenshots (if appropriate)
#### Definition of Done:
